.bumpversion.cfg

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 0.2.4
+commit = True
+tag = True
+message = Prepare v{new_version} release
+
+[bumpversion:file:package.json]
+
+[bumpversion:file:README.md]
+
+[bumpversion:file:plugin.php]
```

Comment:
I’m a bit hesitant to introduce a pip-dependent tool, since that requires having a whole additional programming language’s worth of dependency infrastructure set up on your dev machine. This ought to have been using the version-bump-prompt command internally, and these commands are meant to be run using `npm run bump:…` — to check, did that work, or were you assuming that we use the external bump command? If the `npm run` command doesn't work, we should look at that separately.